lib/components/Widgets/WidgetContainer/index.tsx

```diff
@@ -22,12 +24,19 @@ export interface WidgetContainerProps {
     info?: string
     link?: { title: string; url: string }
   }
+  action?: ButtonProps
+  summaries?: Array<{
+    label: string
+    value: number
+    prefixUnit?: string
+    postfixUnit?: string
```

Comment:
I feel this will eventually become part of a shared type to deal with units, but that's for the future